samples/2-TenantInformation.md

```diff
@@ -0,0 +1,23 @@
+## Tenant information
+
+### Organization contact details
+
+```sh
+mgc organization list
+```
+
+```sh
+mgc organization list --select "DisplayName, City, State, Country, PostalCode, BusinessPhones"
+```
+
+### List application registrations in the tenant
+
+```sh
+mgc organization list --select "DisplayName, id, SignInAudience"
```

Comment:
The command used does not match the title of the snippet - "List application registrations in the tenant".
```suggestion
mgc applications list --select "DisplayName, id, SignInAudience"
```